Добавление третьего задания диплома

Vrazhnov opened the pull request:
Добавлена вся работа по 3 проекту диплома:
Тесты по регистрации пользователя
Тесты по логину пользователя
Тесты по логауту пользователя
Тесты по переходам по кнопкам
Тесты по выбору разделов конструктора
Библиотеки:
allure.version 2.15.0
junit 4.13.2
maven-surefire-plugin 2.22.2
selenium-java 3.141.59

Commit:
Добавление третьего задания диплома

nikolay-977 commented on src/test/java/ru/yandex/test/ConstructorTest.java:
```diff
@@ -0,0 +1,60 @@
+
+    @Before
+    public void setup() {
+        ChromeOptions options = new ChromeOptions();
```
Необходимо исправить: тесты должны запускаться в двух браузерах: Google Chrome и Яндекс.Браузер.
Допустимы реализации: через параметр (файл .properties, переменная окружения, системная проперти, комманд лайн флаг), либо паттерн фабрика.

nikolay-977 commented on src/test/java/ru/yandex/test/LoginTest.java:
```diff
@@ -0,0 +1,114 @@
+    @Before
+    public void setup() {
+/*        ChromeOptions options = new ChromeOptions();
+        System.setProperty("webdriver.chrome.driver", "C:\\Drivers\\WebDriver\\bin\\chromedriver.exe");
```
Необходимо исправить: удали лишний комментированный код

nikolay-977 reviewed (commented):
Можно улучшить: хорошей практикой является добавление README.md с описанием проекта, которое включает название, технологии в проекте (например, Java 11, JUnit 4.13.2, maven 3.9.0 и т.д.), настройка (если необходима) и как запускать (`mvn clean test` - кавычки позволяют выполнять запуск из фалйа README.md). Как оформлять README.md можно ознакомиться в статье https://blog.jetbrains.com/idea/2022/08/markdown-support-improvements/

Commit:
Исправления замечаний:
1) добавлено описание в README.md
2) выполнена оптимизация (cleanup code, optimize imports и rearrange entries)
3) убран закомментированный код и добавлена паттерн фабрика
